src/pages/trainer/training/edit/components/toolbar/presentational.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {ToolbarButton} from './buttons';
+import {
+  HeaderIcon, BoldIcon, ItalicIcon, CodeIcon, LinkIcon, ImageIcon,
+  BulletedListIcon, NumberedListIcon, QuoteIcon,
+} from './icons';
+
+interface IProps {
+  editor: HTMLTextAreaElement;
+  updateEditor: (editor: HTMLTextAreaElement, caret: string, offset: number) => void;
+}
+
+export const ToolbarComponent = (props: IProps) => {
```

Comment:
The name of the file "presentational", mmm... I think we are not using this approach, what's your take on this? should it be just toolbar and then we have toolbar.container.tsx?